markdown/blog/websocket-part2.md

```diff
@@ -809,6 +810,6 @@ components:
         type: string
         description: Format of each pair is "A/B", where A and B are ISO 4217-A3 for standardized assets and popular unique symbol if not standardized.
         pattern: '[A-Z\s]+\/[A-Z\s]+'
-```
+~~~
```

Comment:
What these characters do?

Reply:
We have used tildes (~~~) inplace of backticks (```), to escape the backticks which are inside. 
Backticks and tildes mean the same - **create a  code block** 
You can view about it [here](https://stackoverflow.com/a/31834381/13098662)